pkg/deploy/generator/resources_dev.go

```diff
@@ -328,7 +328,24 @@ enabled=yes
 gpgcheck=yes
 EOF
 
-yum -y install azure-cli podman podman-docker jq gcc gpgme-devel libassuan-devel git make tmpwatch python3-devel go-toolset-1.16.12-1.module+el8.5.0+13637+960c7771
+yum -y install azure-cli podman podman-docker jq gcc gpgme-devel libassuan-devel git make tmpwatch python3-devel htop openssl-devel
+
+# go-toolset-1.17.X is only available for RHEL 8.6 so let's use the Google binaries to build the Microsoft Go BoringSSL code for the time being
```

Comment:
should we switch our CI VMSS to RHEL 8.6 to continue to use the RHEL Go-toolset?

Reply:
I see that RHEL 8.6 is now deployed in Azure:

```
  {
    "offer": "RHEL",
    "publisher": "RedHat",
    "sku": "8-LVM",
    "urn": "RedHat:RHEL:8-LVM:8.6.2022051813",
    "version": "8.6.2022051813"
  },
```

So if we change our version to `8.6.2022051813` in the Image selection above, we should get RHEL 8.6's go. I suggest we do use this instead of "latest" as then that means we won't accidentally get 8.7 or whatever.

Reply:
Now that 8.6 has been released we can indeed.

Although, having a dependancy between the upstreams and us leaves us open to not being able to react quickly to version bumps and CVE's, so there is a case to be made for building directly from MSFTs FIPS releases IMO.